docs/website/docs/technologies/index.md

```diff
@@ -0,0 +1,12 @@
+# Technologies
+
+IREE at it's core is an integration of multiple technologies including:
```

Comment:
Where do you imagine the line is for including a technology in this list?

I could see most of the acronyms in the diagram at https://github.com/google/iree#architecture-overview getting an entry here.

(I left another comment about page organization, and if the list grows to > 5 items/categories I'd definitely want them merged instead of all having their own pages)

Reply:
Changed to be a glossary for summarizing technology. Probably adding a compilation workflow using the thing Ben brought up in discord.